
uint8 front_left_velocity
uint8 front_right_velocity
uint8 back_left_velocity
uint8 back_right_velocity